cfgov/v1/preprocessed/css/molecules/image-and-text-25-75.less

```diff
@@ -0,0 +1,93 @@
+/* topdoc
+  name: Image and Text 25/75
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-image-and-text-25-75">
+            <div class="m-image-and-text-25-75_image-container">
+                <img src="image">
+            </div>
+            <div class="m-image-and-text-25-75_content">
+                <h1 class="m-image-and-text-25-75_heading">Heading</h1>
+                <p class="m-image-and-text-25-75_body">Body content</p>
+                <a class="m-image-and-text-25-75_link">Link</a>
+            </div>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-image-and-text-25-75
+            .m-image-and-text-25-75_image-container
+              img
+            .m-image-and-text-25-75_content
+              .m-image-and-text-25-75_heading
+              p.m-image-and-text-25-75_body
+              a.m-image-and-text-25-75_link
+  tags:
+    - cfgov-molecules
+*/
+
+.m-image-and-text-25-75 {
+
+    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+
+    &_image-container {
+
+        > img {
+            width: 130px;
+
+            .respond-to-min( @desktop-min, {
+                width: 150px;
+            } );
+        }
+        text-align: center;
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+
+        .respond-to-min( @tablet-min, {
+            text-align: left;
+            float: left;
+            margin-right: unit( @grid_gutter-width / @base-font-size-px, em );
+            margin-bottom: 0;
+        } );
+    }
+
+    &_content {
+        .respond-to-min( @tablet-min, {
+            margin-left: unit( ( 130px + @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+
+        .respond-to-min( @desktop-min, {
+            margin-left: unit( ( 150px + @grid_gutter-width ) / @base-font-size-px, em );
+        } );
+    }
+
+    &_heading,
+    &_body,
+    &_link {
+        /* TODO: Margin is wider than expecting when applied to h2.
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em ); */
+        margin-bottom: 30px;
```

Comment:
For this one, you'd have to break out the heading, because it's a different font-size than the body and link